litmus-portal/graphql-server/graph/schema.resolvers.go

```diff
@@ -104,6 +104,18 @@ func (r *queryResolver) GetScheduledWorkflows(ctx context.Context, projectID str
 	return queries.QueryWorkflows(projectID)
 }
 
+func (r *queryResolver) ListWorkflow(ctx context.Context, projectID string, workflowIds []string) ([]*model.Workflow, error) {
+	if len(workflowIds) == 0 {
```

Comment:
Will it be empty array or `nil`  when the frontend doesn't send a parameter at all?

Reply:
It will work if workflowIDs is not present. Graphql convert it to nil